.nvmrc

```diff
@@ -1 +1 @@
-8
+10
```

Comment:
Had to bump Node version to 10. I use a script that automatically changes my node version via nvmrc and `yarn build` fails with Node 8 because the currently locked version of `microbundle` uses promised based `readfile` calls, which are unsupported by Node 8.